game/controllers/movement_controller.py

```diff
@@ -0,0 +1,42 @@
+from game.controllers.controller import Controller
+from game.common.enums import *
+
+
+class MovementController(Controller):
+
+    def __init__(self):
+        super().__init__()
+
+    def handle_actions(self, world, client):
+        avatar_x = client.avatar.position[1]
```

Comment:
avatar_x should be equal to client.avatar.position[0]
avatar_y should be equal to client.avatar.position[1]